R/version.R

```diff
@@ -43,3 +43,17 @@ version <- function(x) {
     profile
   }
 }
+
+#' @rdname version
+#' @param value Version number to assign to `x`.
+#' @noRd
+"version<-" <- function(x, value) {
+  old <- version(x)
+  new <- value
+  x$profile <- sub(old, new, x$profile, fixed = TRUE)
+  x$resources[[1]]$schema <- sub(old, new, x$resources[[1]]$schema, fixed = TRUE)
+  x$resources[[2]]$schema <- sub(old, new, x$resources[[1]]$schema, fixed = TRUE)
+  x$resources[[3]]$schema <- sub(old, new, x$resources[[1]]$schema, fixed = TRUE)
```

Comment:
In `sub(old, new, x$resources[[1]]$schema, fixed = TRUE)` there is a "copy paste" typo. It should be ` x$resources[[2]]$schema` and ` x$resources[[3]]$schema` respectively.

Exactly to avoid these typo's, I would use purrr::map() here:

```r
x$resources <- purrr::map(x$resources, function(resource) {
  resource$schema <- sub(old, new, resource$schema, fixed = TRUE)
  resource
})
```